modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3113,6 +3114,37 @@ public CodegenModel fromModel(String name, Schema schema) {
         return m;
     }
 
+    protected static void setEnumDiscriminatorDefaultValue(CodegenModel model) {
```

Comment:
minor suggestion: adding a docstring (e.g. 1-liner) explaining what this function does